### PR TITLE
Use simple analyzer on indexes. 

### PR DESCRIPTION
**Pull request by wnojopra**
Using the simple analyzer makes the tokenizer split on any non-alphanumeric character, including underscores.

q=phd
Before:
https://screenshot.googleplex.com/DZCQMNVCm94
After:
https://screenshot.googleplex.com/K0N4abC7Up9

q=female
Before:
https://screenshot.googleplex.com/ANF6NQw36E6
After:
https://screenshot.googleplex.com/7kiQVTQuCma

### CODE REVIEW

File: bigquery/indexer.py [hunk 1]
```diff
@@ -267,8 +267,33 @@ def index_fields(es, index_name, table, sample_id_column):
     for field in fields:
         if field.name == sample_id_column:
             id_prefix = "samples." + id_prefix
+    mappings = {
+        'dynamic': False,
+        'properties': {
+            'description': {
```
Comment (melissachang) [inline bigquery/indexer.py]: nit put name before description

File: bigquery/indexer.py [hunk 2]
```diff
@@ -267,8 +267,33 @@ def index_fields(es, index_name, table, sample_id_column):
...
+                'type': 'text',
```
Comment (melissachang) [inline bigquery/indexer.py]: Might as well use simple analyzer for description also

File: bigquery/indexer.py [hunk 3]
```diff
@@ -267,8 +267,33 @@ def index_fields(es, index_name, table, sample_id_column):
     for field in fields:
         if field.name == sample_id_column:
             id_prefix = "samples." + id_prefix
+    mappings = {
```
Comment (melissachang) [inline bigquery/indexer.py]: // Use simple analyzer so underscores are treated as a word delimiter.
// With default analyzer, searching for "baseline" would not find BQ column named "age_at_baseline".
// With simple analyzer searching for "baseline" would find BQ column named "age_at_baseline".

File: bigquery/indexer.py [hunk 4]
```diff
@@ -363,6 +388,7 @@ def create_mappings(es, index_name, table_name, fields, participant_id_column,
                     'ignore_above': 256
                 }
             }
+            properties[field_name]['analyzer'] = 'simple'
```
Comment (melissachang) [inline bigquery/indexer.py]: // Use simple analyzer so underscores are treated as a word delimiter.
// Underscores in BQ column contents are not as common as underscores in column names, but
// some datasets have them (such as Baseline).

**Comment by wnojopra** [comment]
NHS and UKBB are indexing on the second cluster right now. Let me do the same for 1000_genomes, amp_pd, and both baselines

**Comment by wnojopra** [comment]
FYI second cluster is up for all data explorers, and indexing is complete on all.

**Comment by wnojopra** [comment]
All api servers are re-deployed, and I'm halfway through deleting all the old clusters.

**Pull request merged**